examples/02-Merlin-Models-and-NVTabular-applying-to-your-own-dataset.ipynb

```diff
@@ -3,7 +3,7 @@
   {
```

Comment:
<p>We renamed the valid_data set to test_dataset. So the parquet file is called valid but we call it test dataset.</p><p> </p><p>I think we need to choose, if this is a valid or test dataset.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/262/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>